src/Microsoft.DotNet.Build.Tasks/EvaluateMSBuildProject.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.Build.Utilities;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Framework;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class EvaluateMSBuildProject : Task
```

Comment:
See my other note but right now I really don't like this task and we should try to eliminate it if we can.